S23/mepripri/ToLowerCase.py

```diff
@@ -0,0 +1,49 @@
+import csv
+
+def splitLang(l):
+    l=l.split('-')
+    if len(l[0])==2:
+        l=l[0]
+    return l
+
+def toLower(s,lang):
+    str=''
+    first=0
+    lang=splitLang(lang)
+    if lang=='tr' or lang=='az':
+        for i in s:
+            if ord(i)==73:
+                str+=chr(305)
+            else:
+                str+=i.lower()
+    elif lang=='ga':
+        first=0
+        for i in range(len(s)):
+            if i==0:
+                if (s[0]=='n' or s[0]=='t') and s[1] in ['A','E','I','O','U','Á','É','Í','Ó','Ú']:
+                    str+=s[0].lower()+'-'+s[1].lower()
+                    first=1
+                else:
+                    str+=s[i].lower()
+            elif i==1 and first==1:
```

Comment:
I'm not sure what the `elif` at line 28 is doing. This may be omitted as it doesn't appear to do anything as of this moment.